code/game/objects/items/theft_items.dm

```diff
@@ -82,9 +120,20 @@
 	else
```

Comment:
It won't let me select the line to comment on it, but I'd suggest putting a `cracked` check on line 76/114. Currently if you click on it with the core while it's cracked it just drops it onto the floor under you, which doesn't seem like intended behaviour.
```diff
-/obj/item/nuke_core_container/attackby(obj/item/nuke_core/plutonium/core, mob/user)
-	if(!istype(core))
-		return ..()
+/obj/item/nuke_core_container/attackby(obj/item/nuke_core/plutonium/core, mob/user)
+	if(!istype(core) || cracked)
+		return ..()

Reply:
Done!